### PR TITLE
MES-7576 Manoeuvres schema updates

### PR DESCRIPTION
**Pull request by gittins7**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/test-report/cat-b/components/manoeuvres-popover/manoeuvres-popover.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ interface ManoeuvresFaultState {
   reverseParkCarpark: boolean;
   forwardPark: boolean;
   reverseLeft?: boolean;
+  reverseManoeuvre?: boolean;
```
Comment (matthew2564) [inline src/app/pages/test-report/cat-b/components/manoeuvres-popover/manoeuvres-popover.ts]: Does this need to be here? Looks like you've amended a Cat B component

File: mock/generate-local-journal.ts [hunk 2]
```diff
@@ -1443,6 +1443,74 @@ export const localJournal: ExaminerWorkSchedule = {
...
+      testCentre: {
+        centreId: 54321,
+        centreName: 'Example Test Centre',
+        costCode: 'EXTC1',
+      },
+      vehicleTypeCode: 'D',
+      vehicleSlotTypeCode: 7,
+      examinerVisiting: false,
+    },
   ],
```
Comment (matthew2564) [inline mock/generate-local-journal.ts:1718]: Can we add 3 new entries for C1M, CEM and C1EM please

File: src/store/tests/test-data/cat-manoeuvres/manoeuvres/manoeuvres.cat-cm.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  on(
+    manoeuvresActions.RecordManoeuvresSelection,
+    (state, { manoeuvre }) => ({
+      [manoeuvre]: {
+        ...state[manoeuvre],
+        selected: true,
+      },
+    }),
+  ),
+  on(
```
Comment (matthew2564) [inline src/store/tests/test-data/cat-manoeuvres/manoeuvres/manoeuvres.cat-cm.reducer.ts:27]: There are no driving faults on manoeuvre categories, so we should be able to remove this

File: src/app/pages/test-report/cat-b/components/manoeuvres-popover/manoeuvres-popover.ts (hunk 1 above)
Comment (matthew2564) [reply src/app/pages/test-report/cat-b/components/manoeuvres-popover/manoeuvres-popover.ts]: Update shouldManoeuvreDisable to be

```
shouldManoeuvreDisable(manoeuvre: Exclude<ManoeuvreTypes, ManoeuvreTypes.reverseManoeuvre>): Observable<boolean> {
```

**Review by matthew2564 — APPROVED** [review]

**Review by solomonsimukoko — APPROVED** [review]

**Pull request merged**